docs/grammar/SolidityParser.g4

```diff
@@ -175,6 +175,17 @@ locals[
 	 )*
 	(Returns LParen returnParameters=parameterList RParen)?
 	(Semicolon | body=block);
+
+/** The definition of a free function.
+ *
+ */
+ freeFunctionDefinition:
```

Comment:
I feel like this and the `NumberUnit` changes should be separate PRs, since it's a bit confusing this way, especially since the tests only cover the `NumberUnit` changes.

Reply:
Yeah, would not mind having those as separate PRs, though I'm also fine with one because we already have quite a few suffix-related PRs :)

Reply:
I felt that since we already tested (and cameel reviewed) the changes in the literal suffix PR, it would be easier to just push this as a single one. Plus, not too many changes.
But, no problem splitting it, if we consider that should be done.